scripts/globals/items/bronze_bed.lua

```diff
@@ -8,7 +8,7 @@ local itemObject = {}
 
 itemObject.onFurniturePlaced = function(player)
     if player:getQuestStatus(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.GIVE_A_MOOGLE_A_BREAK) == QUEST_AVAILABLE then
-        player:setCharVar("Quest[4][100]bedPlacedTime", os.time())
+        player:setCharVar("Quest[4][100]bedPlacedTime", getConquestTally())
         player:setLocalVar("Quest[4][100]mustZone", 1)
```

Comment:
Can this be put into the Quest framework instead of the item's lua?  Including the mustZone var as well -- the Quest framework has a mustZone var included [ quest:setMustZone(player) I think? ]

Reply:
Hmmm... actually can it? Would the quest know when you place the bed?

Reply:
You should be able to? You can call mob events in the quest.lua like "onMobDeath". I imagine if you call an itemobject you should be able to call all its respective functions and what have you. worth a shot I'd say

Reply:
I attempted this (in addition to adding the Bronze Bed item id):
```
[xi.items.BRONZE_BED] =
{
    onFurniturePlaced = function(player)
        print("onFurniturePlaced", getConquestTally())
        quest:setVar(player, "bedPlacedTime", getConquestTally())
        quest:setMustZone()
    end,

    onFurnitureRemoved = function(player)
        quest:setVar(player, "bedPlacedTime", 0)
    end,
},
```
I also tried having the index be `['bronze_bed']` and that wasn't successful either.

Any thoughts if I did something wrong/unexpected?

Reply:
if that doesn't work, it exhausts my ideas lol. @paladindamarus ? you got anything?

Reply:
Hmm ... Not off the time.  I'm thinking maybe the interaction framework doesn't have something to handle it yet since they're all based on Zones.  Not sure how it would handle the mog house.  I'd have to play around with it.  

Good enough for now :)